pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/experiment-creation.component.html

```diff
@@ -49,8 +49,18 @@
           </form>
         </mat-step>
 
-        <!--Parameters-->
         <ng-container *ngIf="algorithmForm.value.type === 'hp'; else nasConfig">
+          <!--Early Stopping-->
+          <mat-step [stepControl]="hyperParamsArray">
+            <form [formGroup]="hyperParamsArray">
```

Comment:
Should we change it to.
```suggestion
          <mat-step [stepControl]="earlyStoppingForm">
            <form [formGroup]="earlyStoppingForm">
```

Reply:
Yeap, thanks! Never trust copy paste